portfolio/src/main/java/com/google/sps/data/Comments.java

```diff
@@ -0,0 +1,26 @@
+package com.google.sps.data;
+
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Comments {
+
+  class Comment {
+    private String userName;
+    private String comment;
+    public Comment(String userName, String comment){
+      this.userName = userName;
+      this.comment = comment;
+    }
+  }
+  private final List<Comment> commentlist = new ArrayList<Comment>();
+
+  public void addComment(String userName, String comment) {
+    commentlist.add(new Comment(userName, comment));
+  }
+
+  public List<Comment> getComments() {
+    return this.commentlist;
```

Comment:
Nit: For consistency, either remove `this` on this line or add `this` on line 20.

Reply:
done!